### PR TITLE
Ft web key

### PR DESCRIPTION
**Pull request by summercode18**
Fixes #
Sizing issues on windows for new key updates.
## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/NewKeyMapFrame.java [hunk 1]
```diff
@@ -31,6 +31,7 @@ public NewKeyMapFrame()
         final JFXPanel fxPanel = new JFXPanel();
 
         setSize(748, 610);
+        //setSize(900,700);
```
Comment (brelsftm) [inline open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/NewKeyMapFrame.java]: Remove this if you dont need it

File: open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlUI.java [hunk 2]
```diff
@@ -3,259 +3,280 @@
...
+	/**
+	 * The directory of the "mouse right button clicked" icon.
+	 */
+	private static final String mouseRight = "images/keys/MouseRight.png";
+
+	/**
+	 * The directory of the "four way arrow All" icon.
+	 */
+	private static final String arrowAll = "images/keys/arrows.png";
+	/**
```
Comment (brelsftm) [inline open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlUI.java]: Got some formatting issues

File: open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/KeyMapPlugin.java [hunk 3]
```diff
@@ -1,27 +1,25 @@
 package io.opensphere.core.control.keybinding;
 
+import cern.colt.Arrays;
```
Comment (brelsftm) [inline open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/KeyMapPlugin.java]: I dont think this is needed, unused?

File: open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlUI.java [hunk 4]
```diff
@@ -3,259 +3,280 @@
...
+
+	/**
+	 * Creates the main pannel for the "Globe Controls" tab. Use of column and row
+	 * constraints to ensure proper parent fitting.
+	 * 
+	 * @param width  the horizontal size desired.
+	 * @param height the vertical size desired.
+	 */
+	public ControlUI(int width, int height) {
+		// setMinSize(width, height);
```
Comment (brelsftm) [inline open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlUI.java]: Remove comments

File: open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlTab.java [hunk 5]
```diff
@@ -18,168 +18,173 @@
...
+		final Label label = new Label(text.getFontCode());
+		label.setTextFill(Color.WHITE);
+		label.setOpacity(75);
+
+		final String fontName = "icons-" + text.getFont().getFontName().replaceAll("\\s", "-");
+		label.getStyleClass().addAll(fontName);
+		label.setStyle("-fx-font-size: 16 px;");
+
+		customIcon.setGraphic(label);
+		customIcon.setStyle("-fx-background-color:#0066cc;-fx-opacity: 1;");
```
Comment (brelsftm) [inline open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlTab.java]: This should be a constant string, its used in 3 places in the file

File: open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlTab.java [hunk 6]
```diff
@@ -18,168 +18,173 @@
...
+		return styledButton;
+	}
+
+	/**
+	 * A custom button for putting text or images into buttons.
+	 */
+	public class IconDispButton extends Button {
+		public IconDispButton(String key) {
+			setDisable(true);
+			setStyle("-fx-background-color:#0066cc;-fx-opacity: 1;");
```
Comment (brelsftm) [inline open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlTab.java]: hrr

File: open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlTab.java [hunk 7]
```diff
@@ -18,168 +18,173 @@
...
+
+	/**
+	 * creates a custom blue styled button with an image inside.
+	 * 
+	 * @param image the icon to be set.
+	 * @return a Button containing the image with fixed size and matching styles.
+	 */
+	public Button createStyledButton(String image) {
+		Button styledButton = new Button();
+		styledButton.setStyle("-fx-background-color:#0066cc;-fx-opacity: 1;");
```
Comment (brelsftm) [inline open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlTab.java]: here

File: open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlTab.java (hunk 5 above)
Comment (summercode18) [reply open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlTab.java]: replaced with fxTheme

File: open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/NewKeyMapFrame.java [hunk 8]
```diff
@@ -31,6 +31,7 @@ public NewKeyMapFrame()
         final JFXPanel fxPanel = new JFXPanel();
 
         setSize(748, 610);
+        // setSize(900,700);
```
Comment (brelsftm) [inline open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/NewKeyMapFrame.java]: Remove this if you dont need it

**Pull request merged**